tests/chainer_tests/test_optimizer.py

```diff
@@ -86,9 +86,9 @@ def setUp(self):
     def setup_cpu(self):
         self.optimizer.setup((self.params, self.grads))
 
-    def setup_gpu(self):
-        self.params = list(map(cuda.to_gpu, self.params))
-        self.grads = list(map(cuda.to_gpu, self.grads))
+    def setup_gpu(self, dst_id):
```

Comment:
I think that `dst_id = None` is better.
If `dst_id` is None, assign `cupy.cuda.Device().id` to `dst_id`.